src/utils/isHookCreated.js

```diff
@@ -1,13 +1,14 @@
 // @flow
 import execa from 'execa'
 import fs from 'fs'
+import path from 'path'
 
 import HOOK from '../commands/hook/hook'
 
 const isHookCreated = async () => {
   try {
-    const { stdout } = await execa('git', ['rev-parse', '--absolute-git-dir'])
-    const hookFile = stdout + HOOK.PATH
+    const { stdout } = await execa('git', ['config', '--get', 'core.hooksPath'])
```

Comment:
Shouldn't we check here for the default hooks path if no core.hooksPath is provided?